main.py

```diff
@@ -1,7 +1,41 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Actor, Genre
 
 
 def main() -> QuerySet:
-    pass
+    genres = [
+        "Western",
+        "Action",
+        "Dramma",
```

Comment:
The genre 'Dramma' is likely a typo and should be 'Drama'. This is corrected later in the code, but it's good practice to use the correct spelling initially.